tests/resources/serializers/test_datacite_serializer.py

```diff
@@ -219,10 +219,19 @@ def test_datacite43_serializer(running_app, full_record):
         "sizes": ["11 pages"],
         "formats": ["application/pdf"],
         "version": "v1.0",
-        "rightsList": [{
-            "rights": "Creative Commons Attribution 4.0 International",
-            "rightsUri": "https://creativecommons.org/licenses/by/4.0/",
-        }],
+        "rightsList": [
+            {
+                'rights': 'Creative Commons Attribution 4.0 International',
```

Comment:
This first one I don't think is correct. Second one below is correct. Or is this a custom license? If so, it's perhaps better to use another example.

Reply:
Yes, it's a custom one, I will change the values to make it more clear :+1: